python/aixexplainer/aixserver/model.py

```diff
@@ -50,6 +50,21 @@ def _predict(self, input_im):
 
     def explain(self, request: Dict) -> Dict:
         instances = request["instances"]
+        try:
+            if "top_labels" in request:
+                self.top_labels = int(request["top_labels"]) 
```

Comment:
Can we change this so that it uses any parameters from the request, but does not modify the original parameters?
`top_labels = int(request["top_labels"]) if ("top_labels" in request) else self.top_labels `